test/uk/gov/hmrc/traderservices/journey/TraderServicesFrontendModelSpec.scala

```diff
@@ -397,7 +397,7 @@ class TraderServicesFrontendModelSpec extends UnitSpec with StateMatchers[State]
         }
 
       for (requestType <- ImportRequestType.values)
-        s"go to AnswerImportQuestionsContactInfo when submittedImportQuestionsAnswerFreightType and requestType=${ImportRequestType
+        s"go to AnswerImportQuestionsOptionalVesselInfo when submittedImportQuestionsAnswerFreightType and requestType=${ImportRequestType
```

Comment:
Line 369 - I don't think we need the "filterNot" maritime here anymore and we can probably get rid of one of these two tests as they'll both getting the same outcome now (seems slightly redundant to me but if you can see value in them lemme know & keep em both)